rskj-core/src/main/java/org/ethereum/core/BlockHeader.java

```diff
@@ -506,4 +590,10 @@ public String getShortHashForMergedMining() {
     public byte[] getHashForMergedMining() {
         return HashUtil.sha3(getEncoded(false));
     }
+
+    public static class SealedBlockHeaderException extends RuntimeException {
```

Comment:
Creating this exception on it's own file would be better. That's a way to avoid unnecessary growth on this already too long class file.